node/init.go

```diff
@@ -43,14 +49,13 @@ func Init(path string, tp Type) error {
 		return err
 	}
 
-	cfg := DefaultConfig(tp)
 	cfgPath := configPath(path)
 	if !utils.Exists(cfgPath) {
 		err = SaveConfig(cfgPath, cfg)
 		if err != nil {
 			return err
 		}
-		log.Info("New config is generated")
+		log.Info("Saving config")
```

Comment:
maybe this should print "Config save in /foo/bar/blah/" or sth like that?

Reply:
The user configures where the Repo goes, so he knows where Config will be located.

Reply:
Would still be useful to add it in the log message. In the future tools that use this node may config this automatically and it's good to have paths in log files in that scenario.